libsolidity/ast/Types.cpp

```diff
@@ -2428,7 +2424,7 @@ FunctionType::FunctionType(VariableDeclaration const& _varDecl):
 		if (auto mappingType = dynamic_cast<MappingType const*>(returnType.get()))
 		{
 			m_parameterTypes.push_back(mappingType->keyType());
-			m_parameterNames.push_back("");
+			m_parameterNames.emplace_back("");
```

Comment:
You could even have ``.emplace_back();`` here - but that may in fact look weird, so I'd say it's fine as it is as well.